service/worker/indexer/esProcessor_test.go

```diff
@@ -32,6 +32,9 @@ import (
 	"github.com/stretchr/testify/suite"
 	"go.uber.org/zap"
 
+	"github.com/uber/cadence/.gen/go/indexer"
```

Comment:
This import ordering looks non-standard - make sure your editor is setup to run goimports on save file